manifests/base/controller-manager/controller-config.yaml

```diff
@@ -26,6 +26,10 @@ data:
           maxBytes: -1
           replicas: 3
           duplicates: 300s
+          duplicates:           # 0: Limits, 1: Interest, 2: WorkQueue
```

Comment:
Somehow it's still there... @talebzeghmi

Reply:
fixed, sorry didn't catch the typo!